### PR TITLE
SPLAT-1445: implement public ipv4 pool with CAPI presubmit

### PR DESCRIPTION
**Pull request by rvanderp3**

### CODE REVIEW

**Comment by rvanderp3** [comment]
/test all

cc @mtulio

**Comment by rvanderp3** [comment]
/pj-rehearse pull-ci-openshift-installer-master-altinfra-e2e-aws-ovn-public-ipv4-pool

File: ci-operator/config/openshift/installer/openshift-installer-master__altinfra.yaml [hunk 1]
```diff
@@ -119,6 +119,20 @@ tests:
     cluster_profile: aws
     workflow: openshift-e2e-aws-proxy
   timeout: 6h0m0s
+- always_run: false
+  as: e2e-aws-ovn-public-ipv4-pool
```
Comment (patrickdillon) [inline ci-operator/config/openshift/installer/openshift-installer-master__altinfra.yaml:123]: nit: you might want to throw `capi` into the name. although not a requirement--altinfra should be sufficient

Comment (rvanderp3) [reply ci-operator/config/openshift/installer/openshift-installer-master__altinfra.yaml:123]: hmm .. yeah, good call.  i'll make that change

**Comment by rvanderp3** [comment]
/pj-rehearse pull-ci-openshift-installer-master-altinfra-e2e-aws-ovn-public-ipv4-pool-capi

**Pull request merged**